### PR TITLE
Update A6) to make it clear that stopping the timer always stops the solve.

### PR DESCRIPTION
**Pull request by lgarron**
Sister commit to #579.

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -404,12 +404,12 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - A5a) While inspecting or solving the puzzle, the competitor must not communicate with anyone other than the judge or the WCA Delegate. Penalty: disqualification of the attempt (DNF).
     - A5b) While inspecting or solving the puzzle, the competitor must not receive assistance from anyone or any object other than the surface (also see [Regulation 2i](regulations:regulation:2i)). Penalty: disqualification of the attempt (DNF).
 - A6) Stopping the solve:
-    - A6a) The competitor stops the solve by releasing the puzzle and then stopping the timer. If a stopwatch is in use, the judge stops the stopwatch as soon as the competitor does this.
-        - A6a1) When using a stopwatch as the only timer, the competitor ends the solve by releasing the puzzle and notifying the judge that they have stopped the solve. The judge stops the stopwatch as soon as the competitor does this.
+    - A6a) The competitor releases the puzzle before stopping the solve, and stops the solve by stopping the timer. If a stopwatch is in use, the judge stops the stopwatch as soon as the competitor does this.
```
Comment (pedrosino) [inline wca-regulations.md:407]: This doesn't sound good to me. I prefer the old version. Or something like

The competitor releases the puzzle and then stops the solve by stopping the timer.

File: wca-regulations.md [hunk 2]
```diff
@@ -404,12 +404,12 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
     - A6b) The competitor is responsible for stopping the Stackmat timer correctly.
-        - A6b1) If the timer stops before the end of the solve and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. A competitor forfeits the right to the additional attempt if the WCA Delegate determines that the timer was stopped deliberately.
-        - A6b2) If the timer stops before the end of the solve and displays a time of 0.06 seconds or higher, then the attempt is disqualified (DNF). Exception: if the competitor can demonstrate that the timer malfunctioned, they may receive an extra attempt, at the discretion of the WCA Delegate.
+        - A6b1) If the timer stops before the competitor has solved the puzzle and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. A competitor forfeits the right to the additional attempt if the WCA Delegate determines that the timer was stopped deliberately.
```
Comment (pedrosino) [inline wca-regulations.md:411]: What if I solved the puzzle but didn't stop the timer? (i. e. timer stopped by itself a few moments before I would do it).

File: wca-regulations.md (hunk 1 above)
Comment (AlbertoPdRF) [reply wca-regulations.md:407]: Any of the three versions sounds good to me.

File: wca-regulations.md (hunk 2 above)
Comment (AlbertoPdRF) [reply wca-regulations.md:411]: See A6b2) below.

**Review by AlbertoPdRF — APPROVED** [review]
LGTM!

**Pull request merged**